src/azure-cli/azure/cli/command_modules/rdbms/_util.py

```diff
@@ -35,3 +37,49 @@ def expand(self, dest, model_type, group_name=None, patches=None):
                           validator=get_combined_validator(self.validators))
         else:
             self.argument(dest, options_list=dest_option, arg_type=ignore_type, validator=None)
+
+
+# pylint: disable=inconsistent-return-statements
+def parse_public_network_access_input(public_network_access):
+    # pylint: disable=no-else-return
+    if public_network_access is not None:
+        parsed_input = public_network_access.split('-')
+        if len(parsed_input) == 1 and str(parsed_input).find('.') != -1:
+            return parsed_input[0], parsed_input[0]
+        elif len(parsed_input) == 2:
+            return parsed_input[0], parsed_input[1]
+        else:
+            raise CLIError('incorrect usage: --public/--public_network_access. Acceptable values are \'all\','
+                           ' \'enabled\', \'disabled\', \'<startIP>\' and \'<startIP>-<destinationIP>\' '
```

Comment:
could we expose these values in help meassage? otherwise, customer will fail first and they have to know allowed value from error message.

Reply:
yes, sure. we can update that in the help message.